proto/qdrant/cloud/booking/v1/booking.proto

```diff
@@ -114,6 +114,19 @@ enum PackageStatus {
   PACKAGE_STATUS_DEACTIVATED = 2;
 }
 
+// PackageStatus defines the valid states a package can be in.
+enum PackagePricingTier {
+  // Unspecified pricing tier, necessary default.
+  PACKAGE_PRICING_TIER_STATUS_UNSPECIFIED = 0;
+  // Vanilla standard 2023 tier, which we have ever since Qdrant Cloud.
+  PACKAGE_PRICING_TIER_STANDARD = 1;
+  // Premium tier.
+  // Note: there may be different types of "premium tier" in the database, having different conditions
+  // for the customer. You can see all Underlying `PricingTier`s here: https://admin.cloud.qdrant.io/pricingtier/
+  // Yet the user facing name remains just "Premium" for each potential variation.
```

Comment:
This seems to be an internal note, which should NOT be on a public API [note that this repo is public]